src/Forms/TextInput.tsx

```diff
@@ -0,0 +1,69 @@
+import React, {
+  ReactElement,
+  useContext,
+  ChangeEventHandler, FunctionComponent,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { BaseTheme } from 'Theme';
+import ValidationErrorMessage from './ValidationErrorMessage';
+
+export interface TextInputProps {
+  /** Function to call on change event */
+  onChange: ChangeEventHandler;
+  /** The id of the text input field */
+  id: string;
+  /** The name of the text input field */
+  name: string;
+  /** The placeholder value of the input field */
+  placeholder?: string;
+  /** Specifies the type of input */
+  type?: string;
+  /** The current value in the text input field */
+  value: string;
+  /** The body of the error message, if applicable */
+  errorMessage?: string;
+}
+
+const StyledTextInput = styled.input`
+    border: ${({ theme }): string => (theme.border.light)};
+    display: block;
+    height: 20px;
```

Comment:
Should the height not be controlled by the font size?

Reply:
...speaking of which - should the font size maybe be a prop that can be passed in?